js/bybit.js

```diff
@@ -1648,47 +1833,16 @@ module.exports = class bybit extends Exchange {
                 isUsdcSettled = defaultSettle === 'USDC';
             }
         }
-        let method = undefined;
+        const enableUnifiedMargin = this.safeValue (this.options, 'enableUnifiedMargin');
```

Comment:
@sc0Vu we should be checking `params` too (for every method). 

Check `handleOptionAndParams` we might use if we override it here and change it slightly.  The stock implementation checks for `default+optionName` which does not really apply here I think

Reply:
@carlosmiei like this?:

```JS
let enableUnifiedMargin = this.safeValue (this.options, 'defaultEnableUnifiedMargin', false);
enableUnifiedMargin = this.safeValue2 (params, 'enableUnifiedMargin', 'defaultEnableUnifiedMargin', enableUnifiedMargin);
params = this.omit (params, [ 'enableUnifiedMargin', 'defaultEnableUnifiedMargin' ]);
```

BTW, should we also change this:
```JS
if (type !== 'spot') {
                let defaultSettle = this.safeString (this.options, 'defaultSettle', 'USDT'); // this line seems not work
                defaultSettle = this.safeString2 (params, 'settle', 'defaultSettle', isUsdcSettled); // change to this.safeString2 (params, 'settle', 'defaultSettle', defaultSettle);
                params = this.omit (params, [ 'settle', 'defaultSettle' ]);
                isUsdcSettled = defaultSettle === 'USDC';
            }
```

Reply:
@sc0Vu 

> let enableUnifiedMargin = this.safeValue (this.options, 'defaultEnableUnifiedMargin', false);
> enableUnifiedMargin = this.safeValue2 (params, 'enableUnifiedMargin', 'defaultEnableUnifiedMargin', enableUnifiedMargin);
> params = this.omit (params, [ 'enableUnifiedMargin', 'defaultEnableUnifiedMargin' ]);

⬆️  this is what `handleOptionAndParams` does, but for the `enableUnifiedMargin` option I'm not sure that the `defaultEnableUnifiedMargin` makes sense tbh, that's why I was suggesting overriding handleOptionAndParams here to remove the "default" part, wdyt?

> if (type !== 'spot') {
>     let defaultSettle = this.safeString (this.options, 'defaultSettle', 'USDT'); // this line seems not work
>     defaultSettle = this.safeString2 (params, 'settle', 'defaultSettle', isUsdcSettled); // change to this.safeString2 (params, 'settle', 'defaultSettle', defaultSettle);
>     params = this.omit (params, [ 'settle', 'defaultSettle' ]);
>     isUsdcSettled = defaultSettle === 'USDC';
> }
> 

⬆️  here I think we can use `handleOptionAndParams` directly no? like: `handleOptionAndParams(params, 'method name', 'settle', 'USDT')`

Reply:
@carlosmiei that make sense, I'll update.